content/preferences/index.md

```diff
@@ -98,7 +98,7 @@ These affect open and save operations.
 
 ### Recent Files History
 
-These change how the list of recent files ("MRU") is displayed in the File menu
+These change how the list of recent files (also knows as the most-recently-used list, or "MRU" for short) is displayed in the File menu
 
 * `☐ Don't check at launch time`: will skip checking whether files in the MRU exist at launch time.
```

Comment:
knows -> know**n**

It's okay as you've changed it; if I were doing it I might capitalize for emphasis ("also known as the Most-Recently-Used list"), but really it is fine.

Reply:
fixed